config/post-install/storage-version-migrator.yaml

```diff
@@ -38,3 +38,4 @@ spec:
           image: ko://knative.dev/eventing/vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate
           args:
             - "apiserversources.sources.knative.dev"
+            - "containersources.sources.knative.dev"
```

Comment:
About your attention in the PR, with this post install job, when using the operator this version migration will work, right?

Reply:
yes. 

/cc @houshengbo